analyzer/src/funTest/assets/projects/synthetic/maven-wagon-expected-output.yml

```diff
@@ -29,21 +29,13 @@ project:
         message: "Could not get package information for dependency 'com.example:ftp-artifact:1.0.0':\
           \ ProjectBuildingException: Error resolving project artifact: Could not\
           \ transfer artifact com.example:ftp-artifact:pom:1.0.0 from/to ftp-repository\
-          \ (ftp://localhost:40123): Cannot access ftp://localhost:40123 with type\
-          \ default using the available connector factories: BasicRepositoryConnectorFactory\
+          \ (ftp://localhost:40123): Password not specified for repository ftp-repository\
```

Comment:
commit message / nit: "There Maven can find the correct artifact in this
step and load it into the `ClassRealm` used when resolving dependencies
later."

Does "there" and "in this step" refer to the same thing?

Reply:
You are right, but I'd rather not wait for another CI round to fix the minor wording issue.

Reply:
Agree.